src/reactapp/src/pages/NewEvaluation.js

```diff
@@ -0,0 +1,79 @@
+import {
+    FormControl,
+    FormLabel,
+    FormErrorMessage,
+    FormHelperText,
+    Heading,
+    Select,
+    Input,
+    Box,
+    Button,
+    ButtonGroup,
+    Flex,
+    Stack,
+    useColorModeValue,
+    Text,
+    Checkbox,
+    Link,
+} from "@chakra-ui/react";
+
+export default function NewEvaluation() {
+    return (
+        <Flex minH={"100vh"} align={"center"} justify={"center"}>
+            <Stack spacing={8} mx={"auto"} maxW={"lg"} py={12} px={6}>
+                <Stack align={"center"}>
+                    <Heading fontSize={"6xl"}>New Evaluation</Heading>
+                </Stack>
+                <Box
+                    rounded={"lg"}
+                    bg={useColorModeValue("white", "gray.700")}
+                    boxShadow={"lg"}
+                    alignItems={"center"}
+                    w="100%"
+                    p={8}
+                >
+                    <Stack spacing={4}>
+                        <FormControl>
```

Comment:
When you are ready to bind this form to real data, you'll want to convert it to a Formik form.